app/controllers/expenses_controller.rb

```diff
@@ -44,6 +44,10 @@ def calculate_total
   def destroy
     @expense = Expense.find(params[:id])
     get_user_expenses if @expense.destroy
+    rescue ActiveRecord::RecordNotFound => e
```

Comment:
Useless assignment to variable - `e`.<br>`rescue` at 47, 4 is not aligned with `end` at 54, 2.